clic.adoc

```diff
@@ -3133,6 +3133,128 @@ ID  Interrupt
 2+  other
 ----
 
+== CLIC register access via Memory-Mapped Registers Recommended Format
+If memory-mapped access is provided in addition to access via indirect CSR access, the following regions and access rules are recommended.
```

Comment:
> [...] in addition to access via indirect CSR

Seems like indirect CSR support is still mandatory, meaning that existing (memory-mapped-only) implementations are technically not compliant.
What's the best course of action here? In my head it doesn't make a lot of sense for an implementation to offer _both_ access methods, so maybe have it _either or_, and define two separate extensions e.g. `Smclicmem` or `Smclicind`?